src/box/memtx_tx.c

```diff
@@ -930,10 +930,10 @@ memtx_tx_story_gc()
  */
 static bool
 memtx_tx_story_is_visible(struct memtx_story *story, struct txn *txn,
-			  struct tuple **visible_tuple, bool is_prepared_ok,
-			  bool *own_change)
+			  bool is_prepared_ok, struct tuple **visible_tuple,
```

Comment:
Ditto.

Reply:
Can't catch what do you mean here.

Reply:
I use 'ditto' for sequential comments on the PR page — the previous comment was:

> Please consider using common terminology across our code and transaction specifications, e.g. `is_read_commited_isolation`.